doc/src/manual/strings.md

```diff
@@ -150,14 +150,14 @@ julia> 'x' - 'a'
 23
 
 julia> 'A' + 1
-'B'
+'B': ASCII/Unicode U+0042 (category Lu: Letter, uppercase)
 ```
 
 ## String Basics
 
 String literals are delimited by double quotes or triple double quotes:
 
-```julia
+```jldoctest
```

Comment:
A named doctest would probably be more appropriate here. The other `DocTestSetup`s in this PR can probably also be changed over as well.